src/Libraries/CoreNodes/List.cs

```diff
@@ -1399,7 +1399,7 @@ private static IList IncreaseDepth(IList list, int amt)
         /// </summary>
         private static object DoubleConverter(object obj)
         {
-            if (obj is int)
+            if (obj is int || obj is long)
```

Comment:
is max item the only place where the double converter is used?

Reply:
doubtful

Reply:
Doing a quick search in the code there are only two places in List.cs where DoubleConverter appears (other than the main class definition); MinimumItem and MaximumItem

Reply:
Yes

Reply:
yep yep, just curious if we should be looking for other regressions like this throughout the code base.

Reply:
@mjkkirschner @aparajit-pratap Take a look at QNTM-3423 and all linked issues.